custom-vu/src/main/kotlin/jces1209/vu/page/dashboard/cloud/CloudDashboardPage.kt

```diff
@@ -0,0 +1,25 @@
+package jces1209.vu.page.dashboard.cloud
+
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import jces1209.vu.page.dashboard.DashboardPage
+import org.openqa.selenium.By
+
+class CloudDashboardPage(
+
+    private val jira: WebJira
+) : DashboardPage(
+    driver = jira.driver,
+    uri = jira.base.resolve("/secure/ConfigurePortalPages!default.jspa")) {
+    override fun createDashboard() {
+        driver.findElement(
+            By.xpath("(//button[@data-test-id='directory.dashboards-v2.create-button'])")).click()
```

Comment:
```suggestion
            By.cssSelector("[data-test-id='directory.dashboards-v2.create-button']"))
            .click()
```

Reply:
all test uses xpath, and for me it is more attractive.  i'd leave as is

Reply:
Note that the suggestion also removed the dependency on `button`. In XPath style, it'd look like this:
```suggestion
            By.xpath("(//*[@data-test-id='directory.dashboards-v2.create-button'])")).click()
```